src/reducers/MapReducer.js

```diff
@@ -164,13 +134,20 @@ export default (state = INITIAL_STATE, action) => {
         case INITIALIZE_MAP:
             const {region, sites} = payload;
             const existingMapLoadedState = state.mapLoaded;
+            const selectedSiteIsEmpty = _.isEmpty(state.selectedSite);
+            const lastKnownRegionIncludingSelectedSite = selectedSiteIsEmpty ? region : {
+                longitudeDelta: 0.25,
+                latitudeDelta: 0.25,
+                longitude: state.selectedSite.coordinate.longitude,
+                latitude: state.selectedSite.coordinate.latitude
+            };
 
             return payload ? {
                 ...INITIAL_STATE,
-                lastKnownRegion: region,
+                lastKnownRegion: lastKnownRegionIncludingSelectedSite,
                 sites: sites,
                 filterCriteriaKeys: state.filterCriteriaKeys,
-                displaySites: filterSites({sites}, state.filterCriteriaKeys),
+                displaySites: filterSites({sites, filterResultsScrutinyLoose: state.filterResultsScrutinyLoose}, state.filterCriteriaKeys),
```

Comment:
I think I missed this when I took my first pass at the filter scrutiny (all/any in filtering).